src/styles/blocks/_stopwatch.scss

```diff
@@ -0,0 +1,72 @@
+.stopwatch {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-width: $size;
+  height: $size;
+  border-radius: 15%;
+  box-shadow: 0 0 40px 25px #ddd;
+  border: solid 2px $main-color;
+  background-color: #fff;
+
+  &__seconds {
+    box-sizing: border-box;
+    width: 7px;
+    height: $size / 2 - 30px;
+    border-radius: 4px;
+    background-color: $main-color;
+    position: absolute;
+    left: $size / 2 - 3px;
+    top: 30px;
+    animation: rot 60s steps(60, end) infinite;
+    transform-origin: 5px #{$size / 2 - 30px};
```

Comment:
It's not clear what is 30px here, will be better to create separate variable for this value and define what is it